sky/resources.py

```diff
@@ -34,6 +34,48 @@
 }
 
 
+@dataclasses.dataclass
+class AutostopConfig:
+    """Configuration for autostop."""
+    # enabled isn't present in the yaml config, but it's needed for this class
+    # to be complete.
+    enabled: bool
```

Comment:
In other places, we use `idle_minutes == -1` as disabled. Maybe good to keep the same convention here.

Reply:
I feel like it's too implicit and difficult to understand, so I wanted to avoid it. But we could do it here.